assets/js/components/setup/setup-proxy.js

```diff
@@ -89,6 +89,29 @@ class SetupUsingProxy extends Component {
 			siteHostname,
 		} = this.state;
 		const isRevoked = 'revoked' === context;
+		const isSecondAdmin = isResettable;
+
+		let title;
+		let description;
+		let startSetupText;
+
+		if ( isRevoked ) {
+			title = sprintf(
+				/* translators: %s is the site's hostname. (e.g. example.com) */
+				__( 'You revoked access to Site Kit for %s', 'google-site-kit' ),
+				siteHostname
+			);
+			description = __( 'Site Kit will no longer have access to your account. If you’d like to reconnect Site Kit, click "Start Setup" below to generate new credentials.', 'google-site-kit' );
+			startSetupText = __( 'Sign in with Google', 'google-site-kit' );
+		} else if ( isSecondAdmin ) {
+			title = __( 'Sign in with Google to configure Site Kit' );
+			description = __( 'To use Site Kit, sign in with your Google account. Site Kit Service will guide you through 3 simple steps to complete the connection and configure the plugin.' );
```

Comment:
This line should be updated for consistency.
```suggestion
			description = __( 'To use Site Kit, sign in with your Google account. The Site Kit service will guide you through 3 simple steps to complete the connection and configure the plugin.' );
```